lua/wikis/commons/TeamParticipants/Parse/Wiki.lua

```diff
@@ -110,14 +113,15 @@ function TeamParticipantsWikiParser.parseParticipant(input, date)
 				table.insert(potentialQualifiers, Opponent.readOpponentArgs({type = Opponent.team, template = name}))
 			end)
 		end
+		opponent.players = TeamParticipantsWikiParser._getTBDPlayers(playerNumber)
```

Comment:
I believe the intended place for the TBD players is not in contenders, but rather in rosters that are TBA/partially TBA